index.html

```diff
@@ -340,5 +340,32 @@ <h1>
         </div>
       </div>
     </section>
+    <div class="form-div">
+      <form action="https://formspree.io/f/xbjboedq" method="POST" class="form">
+        <h2>Contact me</h2>
+
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          type="text"
+          maxlength="30"
+          placeholder="Enter your name here"
+          required
+        />
+        <input type="email" placeholder="Enter your email here" required />
+        <textarea
+          name="text-area"
+          id="text-area"
+          cols="20"
+          rows="7"
+          maxlength="500"
+          placeholder="Write your message here"
+        ></textarea>
```

Comment:
- [ ] As stated in the Project requirement and suggested by the previous Reviewer , Please make sure that  text area input  have the `required` attribute to activate the form validation and to enable a user cannot submit the form if one or more of the fields is empty.
![Capture](https://user-images.githubusercontent.com/93598189/192626087-10bc366a-8bae-409e-9096-2b9e6a0bde7c.PNG)

- When I submit the form without message the validation show up on the Form-spree services because you give your text-area with id="text-area" with name attribute, But i recommend  you to add `required ` attribute so it will validate the form before the submit.

Your Form submitted without message Look likes this :
![Capture](https://user-images.githubusercontent.com/93598189/192627724-3ce74da5-cc4d-4d46-827a-c0c478580426.PNG)

Reply:
what should i give the id attribute then?

Reply:
Hello @Oscarj-8  only add `required` attribute , there is no problem with id attribute what i recommend is it's better to validate before we submit the form.

Reply:
fixed it.

Reply:
checkout my last commit, and thank you so much